### PR TITLE
Upgrade to Polymer 3

### LINKED ISSUE #20: Upgrade to Polymer 3
No dependencies

### PR DESCRIPTION
**Pull request by cristinecula**
Fixes #20

### CODE REVIEW

**Comment by cristinecula** [comment]
@megheaiulian @nomego

File: cosmoz-default-tree.js [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+		this.pathStringSeparator = options.pathStringSeparator || '/';
+		this.childProperty = options.childProperty || 'children';
+		this.searchProperty = options.searchProperty || 'name';
+	}
+
+	static _objectValues(obj) {
+		if (!obj) {
+			return [];
+		}
+		return Object.keys(obj).map(key => obj[key]);
```
Comment (nomego) [inline cosmoz-default-tree.js]:
```suggestion
		return Object.values(obj);
```

orv maybe drop the helper completely?

File: cosmoz-default-tree.js [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+	}
+
+	static _sortPathNodes(a, b) {
+		const undefCounter = item => {
```
Comment (nomego) [inline cosmoz-default-tree.js]:
```suggestion
		const undefCounter = item => item === undefined,
```

File: cosmoz-default-tree.js [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+		return Object.keys(tree)
+			.map(key => {
+				const subTree = {};
+				subTree[key] = tree[key];
+				return this._getPathNodes(pathLocator, subTree, pathLocatorSeparator);
+			})
+			.filter(item => {
+				return item && item.length > 0;
+			})
+			.sort(CosmozDefaultTree._sortPathNodes)[0];
```
Comment (nomego) [inline cosmoz-default-tree.js]:
```suggestion
			.sort(this.constructor._sortPathNodes)[0];
```
?

**Review by nomego — APPROVED** [review]

**Comment by cristinecula** [comment]
@nomego @Wurper ready for release!

**Pull request merged**